### ISSUE TITLE
LPIPS gives negative value

### ISSUE THREAD

**Issue by F-Barto** [Lightning-AI/torchmetrics#2315]
### Discussed in https://github.com/Lightning-AI/torchmetrics/discussions/2310

lightning==2.1.3
torchmetrics==1.3.0

---

Hi everyone, 

It seems that [LearnedPerceptualImagePatchSimilarity](https://github.com/Lightning-AI/torchmetrics/blob/c86ae7b9264684a10635834614424c1701dc0f78/src/torchmetrics/image/lpip.py#L40) can returns negative values when it is used as loss function.

From what I understand it comes from the linear layers that don't have their `requires_grad` set to false see [self.lins](https://github.com/Lightning-AI/torchmetrics/blob/c86ae7b9264684a10635834614424c1701dc0f78/src/torchmetrics/functional/image/lpips.py#L317). Which have for effect to fine-tune the lpips' network

One workaround is to only give the network parameters and not the LightningModule's (`self.params()`):
```python
class LitAutoEncoder(LightningModule):

    def __init__(self):
        super().__init__()
        self.network = ConvAutoencoder()

        self.lpips_loss = LearnedPerceptualImagePatchSimilarity(
                net_type='vgg', reduction='mean', normalize=False
        )

    def configure_optimizers(self):
        optimizer = torch.optim.Adam(self.network.parameters(), lr=1e-3)
        return optimizer

```

Please find a reproducible example at: https://gist.github.com/F-Barto/ce8a1a28a6d2afd2f2141026fbb3a605

**Comment by F-Barto** [comment]
It seems removing the lpips parameters from the optimizer has for effect to accumulate the gradients over batches.
They are only reinitialized when entering the validation loop (each drop in the image below corresponds to a batch after a validation epoch):

![image](https://github.com/Lightning-AI/torchmetrics/assets/905829/aa822b4a-8c89-4cc1-97b5-045618183359)

**Comment by SkafteNicki** [comment]
Hi @F-Barto, I tried to give the issue a shot in PR #2326. The fix to me seems very simple by also setting `requires_grad=False` for the linear layers.

**Comment by F-Barto** [comment]
Yep, fixed it for me !
Thank you very much :)
